src/blackhole/logger.hpp

```diff
@@ -79,6 +79,7 @@ class composite_logger_t : public base_logger_t {
         using std::swap;
         swap(d.filter, other.d.filter);
         swap(d.frontends, other.d.frontends);
+        swap(d.exception, other.d.exception);
```

Comment:
Every bug fixes should be proved with test. Can you please write an unit or functional test?